app/dashboard/models.py

```diff
@@ -1065,6 +1078,67 @@ def stats(self):
             (success_rate, 'Success Rate'),
         ]
 
+    @property
+    def get_quarterly_stats(self):
+        user_active_in_last_quarter = False
```

Comment:
We need to get in the habit of adding docstrings that adhere to pep-257 when adding new methods and classes.  This is fine for the moment, but something to consider moving forward.